### ISSUE TITLE
broken dev build

### ISSUE THREAD

**Issue by JustasB** [scidash/neuronunit#201]
@russelljjarvis I noticed the [latest build on dev](https://travis-ci.org/scidash/neuronunit/builds/500500675) is not working. Will you check in a working version?

I'm working on morphology unit tests and I want to be confident my tests will work with the latest NU version -- but the unit tests are failing.

**Comment by russelljjarvis** [comment]
There is a bit of a responsibilty conflict with that branch atm. Rick was performing, a merge into it.

Meanwhile, I have started the process of creating more meaningful unit tests, in the more recently maintained branch `opt_cluster`, where I check the effectiveness of optimization, against all the different backends. It's probably about time, I merged opt_cluster, into dev, but that will temporarily break things even more (for about 24 hours at least, then things could be better).

**Comment by rgerkin** [comment]
All tests passed 3 commits earlier, then 2 commits earlier, Python 2.7 failed but others were still passing.  Those reflected my commits.  Then Russell made minor changes that could not possibly have caused the current failures, and yet the tests are now failing.  So my guess is that it is something upstream, maybe SciUnit.  I will test on my machine.

**Comment by rgerkin** [comment]
Fixed in dfdaa1a1d04509eba067af5f9867196b5bb2ebf2.  

Previously I had changed some tests to set attributes like current amplitudes on construction, rather than as class attributes.  This was done because test instances run in parallel might need different current amplitudes.  @russelljjarvis solved this his own way in his fork, and when I pulled his changes in I refactored to put this solution into the parent class for these tests.  

But `RestingPotentialTest` got left out of this, presumably out of confusion about whether such a test should even have a current.  I decided (a long time ago) that it should, and that the current should be 0, otherwise existing external currents in the model will be applied by default, and it isn't a resting potential anymore.  So in the last commit I applied the same thing to resting potential, but for 0 current (always).  That fixes the unit tests.
